omnigan/tutils.py

```diff
@@ -183,17 +183,23 @@ def normalize_tensor(t):
     return t
 
 
-def get_normalized_depth_t(arr, domain, normalize=False):
+def get_normalized_depth_t(tensor, domain, normalize=False):
     if domain == "r":
```

Comment:
Rename `arr` to tensor as it is misleading on the data  type (+ needs docstrings!)